app/views/layouts/application.html.slim

```diff
@@ -14,6 +14,14 @@ html
 
   - classes = [@browser_as_class, controller_name, action_name].join(" ")
   body class=classes ng-app="MauApp"
+    - if Rails.env == "production"
+      javascript:
+        key ||= Conf.ga_key
+        <!-- Google Tag Manager (noscript) -->
+        <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-M7V3ZMB"
```

Comment:
The tracking id is going to show up in the js code anyway, so I'm not hiding it. I set up an "anti-hacking" filter on missionartistsunited.org GA account which basically excludes all other data that's not from the site.

Reply:
What if we set this up to be `production` or `acceptance`?  is there a "easy" way to setup an acceptance env to prove this out?  not a big deal but thought I'd ask.